pkg/expapi/types.go

```diff
@@ -91,7 +92,27 @@ type HorizontalPodAutoscalerSpec struct {
 	MaxCount int `json:"maxCount"`
 	// Target is the target average consumption of the given resource that the autoscaler will try to maintain by adjusting the desired number of pods.
 	// Currently two types of resources are supported: "cpu" and "memory".
-	Target TargetConsumption `json:"target"`
+	Target ResourceConsumption `json:"target"`
```

Comment:
This is intended to be a single resource only? And is an absolute value that itself must be vertically auto-sized?